tests_integration/test_stage.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+
+import os
+from unittest import mock
+from tempfile import NamedTemporaryFile
+from tests_integration.snowflake_connector import snowflake_session
+from tests_integration.test_utils import (
+    extract_row_from_mock,
+    extract_row_from_snowflake_session,
+    contains_row_with,
+    not_contains_row_with,
+)
+
+# TODO add database when we will have one
+DATABASE_NAME = ""
```

Comment:
Shoudl we generate this as uuid instead of empty one?

Reply:
We will but we have other task for it